app/models/subscription.rb

```diff
@@ -16,4 +17,12 @@ def current_day_in_words
     current_day = Time.now.strftime("%A").downcase
     "send_#{current_day}"
   end
+
+  def over_limit?
+    Subscription.where(user_id: self.user_id).size >= 2
+  end
+
+  def needs_verification?
+    self.active == false
```

Comment:
one line function may be too much. You can simply do 
self.active? or @subscription.active?